deploy-scripts/deploy_functions.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+function getProperty {
+    PROP_KEY=$1
+    PROP_VALUE=`cat gradle.properties | grep "$PROP_KEY" | cut -d'=' -f2`
```

Comment:
FYI, you could use `./gradlew properties --no-daemon -q | grep "^$PROP_KEY:" | awk '{print $2}'` to get the property value correctly. This would also allow you to use calculated properties too.